common/lib/xmodule/xmodule/js/src/sequence/display.js

```diff
@@ -78,10 +77,21 @@
 
         Sequence.prototype.updatePageTitle = function() {
             // update the page title to include the current section
-            var positionLink = this.link_for(this.position);
+            var positionLink = this.link_for(this.position),
+                currentSectionTitle;
 
             if (positionLink && positionLink.data('page-title')) {
-                document.title = positionLink.data('page-title') + this.base_page_title;
+                currentSectionTitle = positionLink.data('page-title');
+            }
+
+            if (!this.base_page_title) {
+                // remove the current section's title (if it exists) to get the base
+                this.base_page_title = document.title.replace(currentSectionTitle, '');
+            }
```

Comment:
Could we explicitly pass the base page title vs. the unit title into separate fields at the time of initialization so that we don't have to derive it from the document title? I'm just worried that this might lead to weird little bugs.